opengrok-indexer/src/main/java/org/opengrok/indexer/history/HistoryGuru.java

```diff
@@ -977,7 +977,7 @@ public void run() {
         repositories.clear();
         newrepos.forEach((_key, repo) -> { putRepository(repo); });
 
-        elapsed.report(LOGGER, "done invalidating repositories");
+        elapsed.report(LOGGER, String.format("done invalidating repositories (%d)", newrepos.size()));
```

Comment:
make it evident that the count is number of repos

Reply:
ok